plugin/src/main/groovy/org/openbakery/XcodeProjectFile.groovy

```diff
@@ -70,11 +70,17 @@ class XcodeProjectFile {
 					project.xcodebuild.productName = config.getString("objects." + target + ".productName")
 				}
 				String type = config.getString("objects." + target + ".productType")
-				if (type.equalsIgnoreCase("com.apple.product-type.app-extension")) {
-					project.xcodebuild.productType = "appex"
-				}
+//				if (type.equalsIgnoreCase("com.apple.product-type.app-extension")) {
+//					project.xcodebuild.productType = "appex"
+//				}
+
```

Comment:
When we build the main iOS application, the extensions/widgets are targets that are built as well. The code iterates over all the available targets and if it finds an extension, it changes the productType to "appex" which should not be true as the main target still is of type "app".